indigo/pipelines/text.py

```diff
@@ -1,6 +1,7 @@
 import re
 import tempfile
 
+from django.utils.translation import ugettext as _
```

Comment:
should this be `gettext` as in bulk_creator?

Reply:
whoops, yes. `ugettext` is deprecated since python 3 and is now the same as `gettext`